spec/unit/analyzer/strings_analyzer_spec.rb

```diff
@@ -238,6 +238,7 @@ def subject
         "en_exclusive"      => "Only in english",
         "example"           => "Lorem Ipsum",
         "123-abc-3e7.text"  => "Hello Storyboards",
+        "123-ab-3e7.text"  => "Hello Storyboards",
```

Comment:
Little thing: could you add a space in front of the `=>` operator to not break the column alignment of the values here?